Simplenote/src/main/java/com/automattic/simplenote/utils/ThemeUtils.java

```diff
@@ -66,16 +66,14 @@ public static void setTheme(Activity activity) {
             }
         }
 
-        switch (PrefUtils.getIntPref(activity, PrefUtils.PREF_THEME, THEME_LIGHT)) {
-            case THEME_AUTO:
-                AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_AUTO_TIME);
-                break;
+        switch (PrefUtils.getIntPref(activity, PrefUtils.PREF_THEME, THEME_SYSTEM)) {
             case THEME_DARK:
                 AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_YES);
                 break;
             case THEME_LIGHT:
                 AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_NO);
                 break;
+            case THEME_AUTO:
             case THEME_SYSTEM:
```

Comment:
keeping THEME_AUTO since it can be saved to prefs if you're coming from a previous version of the app, although we removed support for [MODE_NIGHT_AUTO_TIME](https://developer.android.com/reference/androidx/appcompat/app/AppCompatDelegate#MODE_NIGHT_AUTO_TIME()) since it's deprecated.